### ISSUE TITLE
Support Twitter archives as well?

### ISSUE THREAD

**Issue by NelsonMinar** [blackle/mastodon-data-viewer.py#8]
Hey, thanks for the tool! It's great, just what I needed.

An idea for a new feature: support Twitter archives as well. Conceptually it shouldn't be too hard I don't think, the Twitter archive export is in a sensible schema. I'm imagining merging archives here, so one search shows me both Mastodon and old Twitter posts. Might as well merge multiple Mastodon archives too, to help folks who migrated servers.

If this interests you let me know, maybe I can help implement it.

**Comment by blackle** [comment]
I agree this would be useful. I don't have the time to implement this but if you want to I would accept a PR, as long as it doesn't break the existing default flow.
